dist/ui-toolkit.standalone.js

```diff
@@ -28436,10 +28435,104 @@ module.exports = require('./code/index');
 
 
 },{"./code/index":255}],259:[function(require,module,exports){
+module.exports = require('./views/stepperView.jsx');
+
+
+},{"./views/stepperView.jsx":261}],260:[function(require,module,exports){
+'use strict';
+
+var React = require('react');
+var Button = require('../../../button');
+var Input = require('../../../input');
+
+module.exports = function() {
+
+  return (
+    React.createElement("div", {className: "component-stepper"}, 
+      React.createElement("span", {className: "button-container"}, 
+        React.createElement(Button, {handleClick: this.decrement, disabled: !this.canDecrement()}, this.props.decrementDisplayString)
+      ), 
+
+      React.createElement(Input, {type: "text", key: this.state.value, readOnly: true}, this.state.value.toString()), 
+
+      React.createElement("span", {className: "button-container"}, 
+        React.createElement(Button, {handleClick: this.increment, disabled: !this.canIncrement()}, this.props.incrementDisplayString)
+      )
+    )
+  );
+};
+
+
+},{"../../../button":210,"../../../input":237,"react":197}],261:[function(require,module,exports){
+'use strict';
+
+var React = require('react');
+
+module.exports = React.createClass({displayName: "exports",
+
+  propTypes: {
+    value: React.PropTypes.number,
+    onChange: React.PropTypes.func,
+    minValue: React.PropTypes.number,
+    maxValue: React.PropTypes.number,
+    incrementDisplayString: React.PropTypes.string,
+    decrementDisplayString: React.PropTypes.string
+  },
+
+  getDefaultProps: function() {
+    return {
+      value: 0,
+      incrementDisplayString: '+',
+      decrementDisplayString: '-',
+      onChange: function() {}
+    };
+  },
+
+  getInitialState: function() {
+    return {
+      value: this.props.value
+    };
+  },
+
+  decrement: function() {
+    if (!this.canDecrement()) return;
+    var value = this.state.value - 1;
+    this.setState({ value: value });
+    this.props.onChange(this.state.value);
```

Comment:
I think we can simplify this, get rid of the state and just `this.props.onChange( this.props.value - 1 )` but I'm going to check this now.
Also I think we might want to put in a step size option, though as that's not a requirement now we should leave that until it is.